modules/common/test/helpers/volumes.go

```diff
@@ -50,3 +50,80 @@ func (tc *TestHelper) AssertVolumeMountExists(name string, subPath string, volum
 
 	return exist
 }
+
+// AssertVolumeMountPathExists - Returns true if mountPath and subPath exist in
+// the volumeMounts array.
+// Assumptions:
+//   - an empty []corev1.VolumeMount results in a failure
+//   - an empty "name" results in a failure
+//   - when no mountPath and no subPath are passed (empty string) it results in a
+//     failure
+//   - if no mountPath is provided it is skipped during the evaluation and
+//   - if no subPath is provided it is skipped during the evaluation
+//   - when both mountPath and subPath are passed, it returns true only when
+//     both are found in the volumeMount array
+func (tc *TestHelper) AssertVolumeMountPathExists(
+	name string,
+	mountPath string,
+	subPath string,
+	volumeMounts []corev1.VolumeMount,
+) bool {
+	// Early return for an empty volumeMounts list
+	if len(volumeMounts) == 0 {
+		gomega.Expect(false).To(gomega.BeTrue(), "Volume mounts list is empty")
+		return false
+	}
+
+	// Early return for an empty VolumeMount name
+	if name == "" {
+		gomega.Expect(false).To(gomega.BeTrue(), "Volume name cannot be empty")
+		return false
+	}
+
+	// Early return if both mountPath and subPath are empty strings
+	if mountPath == "" && subPath == "" {
+		gomega.Expect(false).To(gomega.BeTrue(), "Both mountPath and subPath are empty")
+		return false
+	}
+
+	// init two bool variables to represent the status of mountPath and
+	// subPath check: when the function reaches this point we have at least an
+	// input that should be checked
+	mountPathExists := false
```

Comment:
could simplify with the below condition to `mountPathExists := mountPath == ""` ?

Reply:
Done